avaframe/com1DFAPy/com1DFACfg.ini

```diff
@@ -67,9 +67,13 @@ R = 0.05
 B = 4.13
 
 #++++++++++++ Resistance force parameters
-hRes = 1
+hRes = 10
+cw = 0.5
+dRes = 0.3
+sres = 5
```

Comment:
Something we need to discuss: maybe have a few presets available (this here is wood) 
Also see general comment -> more descriptive